webrtc-stats.html

```diff
@@ -4209,6 +4218,331 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="encoding-parameters-dict*">
+        <h3>
+          <dfn>RTCEncodingParameterStats</dfn> dictionary
+        </h3>
+        <div>
+          <pre class="idl">dictionary RTCEncodingParameterStats : RTCStats {
```

Comment:
DOMString outboundRtp;

Reply:
OR add encodingParameterId to RTCOutboundRtpStreamStats

Reply:
I favor the second